src/Speckle.Objects/Annotation/Text.cs

```diff
@@ -44,7 +44,7 @@ public class Text : Base
   /// <summary>
   /// Plane will be null if the text object orientation follows camera view
```

Comment:
update this comment to indicate that plane axis vectors will be ignored if screenOriented is true